.env.test

```diff
@@ -0,0 +1 @@
+DIAPER_KEY="testapikey"
```

Comment:
Thoughts on stubbing this value out in our test setup? Might make it less reliant on this new file to run.

Reply:
@edwinthinks That's certainly an option. However, I usually take advantage of the `.env.test` file for this purpose since it's designated for storing environment variables related to application implementation details that are being tested. It's like mimicking the production app's environment variables in tests and serves as a central config specifically for mirroring production variables. While the `spec_helper` typically contains configurations for managing and running specs and testing features. So it felt more natural separating these concerns.

What are your thoughts?

Reply:
Also, referencing the `dotenv` [docs](https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use), this file is specifically intended for use with tests and can serve as a source for other env vars being tested against in the future.

Reply:
@samudary good point! Giving it some thought `spec_helper` can get unwieldy with a bunch of env variables if we aren't careful. I think using it this way sounds good!